src/components/Sidebar.jsx

```diff
@@ -1,7 +1,10 @@
 import styled from 'styled-components'
+import getPlaylist from '../api/playlists';
 import { GenericIcon } from './GenericIcon';
 
 export default function Sidebar() {
+
+    const playlist = getPlaylist();
```

Comment:
this is a collection of playlists, so it should be in plural
```suggestion
    const playlists = getPlaylists();
```